cloudbuild_release.yaml

```diff
@@ -21,30 +21,60 @@ steps:
     args:
       - 'pull'
       - 'gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
-    id: 'pull-image'
+    id: 'pull gcp-variant-transforms image'
```

Comment:
hyphenate the id names in this file instead of spaces (to match cloudbuild_CI.yaml)